### PR TITLE
WDPT Ber Patrick

### PR DESCRIPTION
**Pull request by Patrick-Groot**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,9 +36,17 @@ const tweetsArray = [
 
 function App() {
   return (
-    <div className="App">
-      <Tweet />
-    </div>
+    <>
+      <div className="App">
```
Comment (trebeljahr) [inline src/App.js:40]: why the extra surrounding divs?

File: src/components/Tweet.js [hunk 2]
```diff
@@ -1,34 +1,20 @@
-function Tweet() {
+import Actions from "./Actions";
+import Message from "./Message";
+import ProfileImage from "./ProfileImage";
+import Timestamp from "./Timestamp";
+import User from "./User";
+
+function Tweet(props) {
```
Comment (trebeljahr) [inline src/components/Tweet.js:7]: try destructuring props! 
you only need user, timestamp and message. image you can still get by user.image ^^

File: src/components/User.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+function User(props) {
```
Comment (trebeljahr) [inline src/components/User.js:1]: again, give destructuring a try :)

**Review by trebeljahr — COMMENTED** [review]
good job, try out some more with destructuring :blush:

File: src/components/Tweet.js (hunk 2 above)
Comment (Patrick-Groot) [reply src/components/Tweet.js:7]: > good job, try out some more with destructuring blush

Thanks for the feedback. Nowadays I do destructuring! :D

**Pull request closed without merging**